langlib/lang.function/src/main/ballerina/function.bal

```diff
@@ -21,6 +21,23 @@ import ballerina/jballerina.java;
 # If the arguments specified in `args` are not of the type required by `func`,
 # then this will panic.
 #
+# ```ballerina
+# function fn = getGreeting;
+# // Call `getGreeting` without arguments.
+# any|error greeting = function:call(fn);
+#
+# // Call `getGreeting` with an argument.
+# any|error greetingWithName = function:call(fn, "David");
```

Comment:
We can change the return type to `string` in here.

Reply:
Won't compile.